### ISSUE TITLE
Unable to run Automous Hegician in Mac, cannot create DB

### ISSUE THREAD

**Issue by aravintp** [8ball030/AutonomousHegician#46]
![image](https://user-images.githubusercontent.com/36691872/101973617-52535d00-3c74-11eb-9f4f-ae8bd02bf743.png)
![image](https://user-images.githubusercontent.com/36691872/101973639-7f077480-3c74-11eb-92f6-73476a85853e.png)

We are unable to run Automous Hegician in Mac. Error occurs when its creating database.
FATAL:  no pg_hba.conf entry for host "172.20.0.1", user "admin", database "cortex", SSL off

Not sure how to move forward.

**Comment by aaugustin4omnis** [comment]
I can confirm I was getting a problem with the DB as well (not sure if the exact same).

Manually removed the Docker images and installed from scratch - this threw some problems when it started doing the tests (file with errors attached), but then starting it again and running live seemed to work OK. 

[errors.pdf](https://github.com/8ball030/AutonomousHegician/files/5688258/errors.pdf)

However, running tests manually this time, it failed only 1:

======================================================================
FAIL: test_does_ah_expire_otm_eth_put_option (__main__.TestOptionExecutionTester)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/tests.py", line 213, in test_does_ah_expire_otm_eth_put_option
    self.assertTrue(self.await_order_status_code(EXPIRED, new_order))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 10 tests in 121.296s

FAILED (failures=1)

------------------------

**Comment by 8ball030** [comment]
@aravintp @aaugustin4omnis 

Thanks for bringing this to my attention.

Definitely some issues with mac builds.

@aravintp  Can I ask you to check whether you are able to connect to the database externally?

@aaugustin4omnis  I am 98% sure your issue related to timeouts in the test script. Hit me up on discord so we can have a quick check?

**Comment by aaugustin4omnis** [comment]
@8ball030 that's right - it seemed a timeout on connection to the DB. Tried again today, failed with the same error and then I gave some minutes to Docker to finish itself starting up and run all tests again which all completed successfully! Thanks :)

**Comment by 8ball030** [comment]
@aaugustin4omnis Sweet, glad to hear it.

Can you confirm the order, so we we can figure out where to add a check?

**Comment by 8ball030** [comment]
Also, @aaugustin4omnis if we can open a seperate issue for this, so we can reference it in the fix.

**Comment by aravintp** [comment]
@8ball030 Thank you for reply!

But i'm not sure what database im supposed to connect externally.

Am i supposed to have a database running locally?
Sorry feel like im missing some steps here, please advise.

Thank you!
